c10/test/util/complex_test_common.h

```diff
@@ -384,8 +384,7 @@ C10_HOST_DEVICE void test_arithmetic_assign_complex() {
   static_assert(x2.imag() == scalar_t(3), "");
   constexpr c10::complex<scalar_t> x3 = p(scalar_t(2), scalar_t(2), 1.0_id);
   static_assert(x3.real() == scalar_t(2), "");
-#if !defined(__CUDACC__)
-  // The following is flaky on nvcc
+#if !defined(__CUDACC__) || CUDA_VERSION >= 11020  // compiler bug
```

Comment:
could you add a note explaining what the bug is and that it should be fixed 11.2 onwards

Reply:
Added.